plugin/trino-ignite/src/main/java/io/trino/plugin/ignite/IgniteClient.java

```diff
@@ -373,7 +374,7 @@ public JdbcOutputTableHandle beginCreateTable(ConnectorSession session, Connecto
             columnNames.add(columnMetadata.getName());
             columnTypes.add(columnMetadata.getType());
         }
-        String sql = buildCreateSql(schemaTableName, columns.build(), tableMetadata.getProperties());
+        String sql = buildCreateSql(schemaTableName, columns.build(), columnNames.build(), tableMetadata.getProperties());
```

Comment:
The both `columnNames` and `primaryKeys` are built twice. Could you change the arguments of `buildCreateSql` to `SchemaTableName schemaTableName, List<String> columns, List<String> primaryKeys`?

Reply:
@ebyhr 
Did you mean like `private String buildCreateSql(SchemaTableName schemaTableName, List<String> columns, List<String> primaryKeys)`? 

if we do this, its a bit hard to check for existence of primaryKey in columns as columns contains columnName+type (eg: `name varchar(25)`).
we will need to pass either table properties or columnNames into buildCreateSql.

Reply:
>The both columnNames and primaryKeys are built twic

Yes. we can avoid this step by calling build after 
```
for (ColumnMetadata columnMetadata : tableMetadata.getColumns()) {
            columns.add(getColumnDefinitionSql(session, columnMetadata, columnMetadata.getName()));
            columnNames.add(columnMetadata.getName());
            columnTypes.add(columnMetadata.getType());
        }
```

Reply:
You can check the column existence inside of `beginCreateTable()` instead.

Reply:
done.